File: BackEnd-MyTools/src/main/java/com/example/BackEnd_MyTools/Entitys/Mastery.java
package com.example.BackEnd_MyTools.Entitys;

import java.util.List;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import lombok.Data;

@Document(collection = "Mastery") // MongoDB collection name
@Data
public class Mastery {
    @Id
    public String id;

    public String masterId;
    public String masterName;
    public String masterPhone;
    public String title;
    public int masteryTypeId;
    public String masteryStatuId;
    public String pricingType;
    public int price;
    public String city;
    public int experienceYears;
    public String description;
    public List<String> photoUrls;
}

File: Frontend-MyTools/src/models/Mastery.ts
export interface Mastery {
  id: string;
  masterId: string;
  masterName?: string;
  masterPhone?: string;
  title: string;
  typeId?: number | string;
  masteryTypeId?: number | string;
  masteryStatuId?: string;
  pricingType?: string;
  price: number | string;
  city?: string;
  experienceYears?: number;
  description: string;
  photoUrls: string[];
}
